assets/src/edit-story/components/panels/animation/panel.js

```diff
@@ -50,6 +50,11 @@ import EffectChooserDropdown from './effectChooserDropdown';
 
 const ANIMATION_PROPERTY = 'animation';
 
+const backgroundAnimationTooltip = __(
+  'The bg image is too small to animate. Double click on the bg & scale the image before applying the animation.',
```

Comment:
I doubt users and translators necessarily know what "bg" means here or how it could be translated. Let's write "background"